### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Metsanpeitto**
This is the final merge of the development of the app. In it, all the features have been implemented. 
 - Set up the repository and tools.
 - Created  models according to the ERD diagram. 
 - Created validations and tests.
 - Implemented authentication and authorization.
 - Created the page to add a category.
 - Created the home page.
 - Created the page to add a transaction.
 - Created the transactions page.
 - The app is tested adequately.
 - Deployed on Heroku.
 - Recorded video introducing the features and highlights of the development process.
 - Created a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+The Budget app keeps track is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+Features: 
+- Register and log in, so that the data is private to them.
+- Introduce new transactions associated with a category.
+- See the money spent on each category.
+
+### Video introduction
+
+![video] (https://drive.google.com/file/d/1Ju8bL0QMtSG0qKpDTbWxa3Jkqyn0-tOR/view?usp=sharing)
```
Comment (DeliceLydia) [inline README.md:14]: ## Optional
- [ ] I suggest making it just a link in your README it will make it clean like [video](link)

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @Metsanpeitto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: my highlights for your project:
- [ ] Application is working as eexpected
- [ ] README is professional
- [ ] Demo of your project shows that you understand what you did and you mentioned something you implemented that you are proud of
- [ ] I like that you tested you application

you nailed it congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Metsanpeitto) [reply README.md:14]: Good point. I will do it. :+1:

**Comment by Metsanpeitto** [comment]
Thanks a lot @DeliceLydia !  :+1:

**Pull request merged**